test/integration/multi-node-8/controls/all_spec.rb

```diff
@@ -0,0 +1,11 @@
+control 'all' do
+  %w(11 12).each do |suff|
+    describe host("192.168.60.#{suff}") do
+      it { should be_reachable }
+    end
+  end
+
+  describe host('192.168.60.14') do
+    it { should_not be_reachable }
+  end
+end
```

Comment:
Please make a symlink from ``test/integration`` for ``multi-node-7 -> multi-node-8`` instead of copying these files.